core/build.gradle.kts

```diff
@@ -98,6 +98,7 @@ dependencies {
     testImplementation(kotlin("stdlib-jdk8"))
     testImplementation(kotlin("test"))
     testImplementation(kotlin("test-junit5"))
+    testRuntimeOnly("org.locationtech.proj4j:proj4j-epsg")
```

Comment:
add a comment that proj4j-epsg must not be converted 'implementation' due to its license